api/src/main/java/org/openmrs/Condition.java

```diff
@@ -0,0 +1,211 @@
+	 * @param additionalDetail additional details of the condition to be set
+	 * @param onsetDate the date the condition is set
+	 */
+	public Condition(CodedOrFreeText condition, ConditionClinicalStatus clinicalStatus,
+		ConditionVerificationStatus verificationStatus, Long previousVersion, String additionalDetail,
+		Date onsetDate) {
+		this.condition = condition;
+		this.clinicalStatus = clinicalStatus;
+		this.verificationStatus = verificationStatus;
+		this.previousVersion = previousVersion;
+		this.additionalDetail = additionalDetail;
+		this.onsetDate = onsetDate;
+	}
+
+	/**
+	 * Gets the condition id 
+	 * 
+	 * @return An integer with the conditionId
+	 */
+	public Integer getConditionId() {
+		return conditionId;
+	}
+
+	/**
+	 * Sets the condition id
+	 * 
+	 * @param conditionId the id of the codition to be set
+	 */
+	public void setConditionId(Integer conditionId) {
+		this.conditionId = conditionId;
+	}
+
+	/**
+	 * Gets the condition that has been set
+	 * 
+	 * @return A CodedOrFreeText object that defines the condition
+	 */
+	public CodedOrFreeText getCondition() {
+		return condition;
+	}
+
+	/**
+	 * Sets the condition
+	 * 
+	 * @param condition the condition to be set
+	 */
+	public void setCondition(CodedOrFreeText condition) {
+		this.condition = condition;
+	}
+
+	/**
+	 * Gets the clinical status of the condition
+	 * 
+	 * @return A ConditionClinicalStatus object that defines the clinical status
+	 */
+	public ConditionClinicalStatus getClinicalStatus() {
+		return clinicalStatus;
+	}
+
+	/**
+	 * Sets the clinical status of the condition
+	 * 
+	 * @param clinicalStatus the clinical status of the condition to be set
+	 */
+	public void setClinicalStatus(ConditionClinicalStatus clinicalStatus) {
+		this.clinicalStatus = clinicalStatus;
+	}
+
+	/**
+	 * Gets the verification status of the condition
+	 * 
+	 * @return A ConditionVerificationStatus object that defines the verification status of the condition
+	 */
+	public ConditionVerificationStatus getVerificationStatus() {
+		return verificationStatus;
+	}
+
+	/**
+	 * Sets the verification status of the condition
+	 * 
+	 * @param verificationStatus the verification status of the condition to be set
+	 */
+	public void setVerificationStatus(ConditionVerificationStatus verificationStatus) {
+		this.verificationStatus = verificationStatus;
+	}
+
+	/**
+	 * Gets the previous version of the condition
+	 * 
+	 * @return A condition object showing the previous version of the condition
+	 */
+	public Long getPreviousVersion() {
+		return previousVersion;
+	}
+
+	/**
+	 * Sets the previous version of the condition
+	 * 
+	 * @param previousVersion the previous version of the condition to be set
+	 */
+	public void setPreviousVersion(Long previousVersion) {
+		this.previousVersion = previousVersion;
+	}
+
+	/**
+	 * Gets the addition detail of the condition
+	 * 
+	 * @return A string showing additional detail of the condition
```

Comment:
Make the comment sentence start with lower case for @return

Reply:
Noted

Reply:
Done